doc/source/data/doc_code/hf_quick_start.py

```diff
@@ -2,6 +2,43 @@
 # isort: skip_file
 # fmt: off
 
+# __hf_super_quick_start__
+import ray
+import numpy as np
+import pandas as pd
+from typing import Dict
+
+ds = ray.data.from_pandas(pd.DataFrame(["Complete this", "for me"], columns=["text"]))
+
+class HuggingFacePredictor:
+    def __init__(self):
+        from transformers import pipeline
+        self.model = pipeline("text-generation", model="gpt2")
+
+    def __call__(self, batch: Dict[str, np.ndarray]):
+        model_out = self.model(list(batch["text"]), max_length=20)
+        return pd.DataFrame({"output": model_out})
```

Comment:
The `__call__` function returns pandas in this HuggingFace example, but returns numpy in the Torch/TF examples. New readers might be confused about this. 
And even within the HF example, the `__hf_super_quick_start__` part reads data from pandas, while the `__hf_no_ray_start__` part reads data from numpy.
 Maybe we should stick to numpy in the quick start to avoid confusion. Later we can explain we support more than numpy data formats.

Reply:
makes sense, thanks